assignment.js

```diff
@@ -32,6 +32,14 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let n = 0
+
+while (n <= 12) {
+  console.log(n)
+  n++
+  n*2
+}
+
 // Add your code above this line
 
```

Comment:
@jl1982 Good job utilizing the while loop to solve this problem. Good job with your indentation on this problem. take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L31)